### DISCUSSION TITLE
Roadmap for Zed code editor

### DISCUSSION THREAD

**Discussion by theArianit** [zed-industries/zed#6470] [Feature Requests]
Hello. I am a linux user. Since Zed is currently only available for macOs, it would be nice to have some kind of a roadmap. I am counting every grain of my sand watch until I can try Zed out. 
Is there any plan to make a roadmap? When can we expect Zed to be available for other platforms too?
Thanks

**Comment by widersky** [comment]
AFAIK for now Zed is coded only for macOS - versions for Windows and Linux will appear closer to the release of version 1.0, which is probably in a few good months. In my opinion, this is a good approach, because by focusing on one platform, the authors are able to deliver specific functionalities without bothering in compatibility with each OS. When the functionality of version 1.0 is frozen, then they can focus on adapting the editor to other platforms. And then develop everything in parallel.

**Reply by JosephTLyons** [reply]
This answer is mostly correct. The only tweak I'd make here is we haven't entirely settled on when add support for other platforms.  It's on our roadmap, but in an area that isn't well-defined.  It won't happen before we go into our beta phase, which will actually be happening fairly soon :).

**Comment by JosephTLyons** [comment]
Just to update this old Discussion, we do have a public roadmap for 2024:

- https://zed.dev/roadmap

As for platforms, both Linux and Windows are being worked on right now. Linux can be compiled from source and binaries can be downloaded from each GitHub Release. Windows can also be compiled. Note that neither of these are considered "complete" or "stable" yet.
